Automattic/jetpack
# Instant Search: refine overlay transition

#### Changes proposed in this Pull Request:
* Adds a customizer option to enable/disable the overlay transition.
* Speeds up the transition and makes it smoother by just using transition for its opacity.

![jps-transition](https://user-images.githubusercontent.com/390760/76885245-d5c35780-6876-11ea-9e47-089f37ea0ae4.gif)

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* No.

#### Testing instructions:
1. Follow [these setup instructions](https://github.com/Automattic/jetpack/blob/instant-search-master/modules/search/instant-search/README.md#testing-instructions) to set up Jetpack Instant Search on your site.
1. Enter a search term to make the overlay appear. Press ESC or close the overlay manually. Test to see if the animation feels good on both scenarios.
1. Go to the Customizer and disable the fade in/out.
1. Enter a search term and ensure the overlay no longer has an animation.

#### Proposed changelog entry for your changes:
* None.
